File: doc/_i18n/jp/tutorials/introduction/dual-arm-controller/add_collision_constraint.cpp
// コンストラクタ内、またはreset関数内
double iDist = 0.1;
double sDist = 0.05;
double damping = 0.0;
addCollisions(
  "ur5e", "kinova_default", {{"*", "*", iDist, sDist, damping}}
);

File: doc/_i18n/jp/tutorials/introduction/dual-arm-controller/add_collision_constraint.py
# コンストラクタ内、またはresetコールバック関数内
iDist, sDist, damping = 0.1, 0.05, 0.0
self.addCollisions(
    "ur5e",
    "kinova_default",
    [mc_rbdyn.Collision("*", "*", iDist, sDist, damping)],
)
